source/_integrations/hdmi_cec.markdown

```diff
@@ -18,6 +18,12 @@ The `hdmi_cec` integration provides services that allow selecting the active dev
 
 ## CEC Setup
 
+### Home Assistant OS
+
+To test if hdmi-CEC will work on your Home Assistant OS installation you can use the official "CEC Scanner" Add-on. Run this add-on to discover if your hardware has HDMI-CEC capabilities and which devices are connected. Do not have this add-on `Start on boot` as it will interfere with the integration. 
```

Comment:
```suggestion
To test if HDMI-CEC will work on your Home Assistant OS installation, you can use the official **CEC Scanner** add-on. Run this add-on to see if your hardware has HDMI-CEC capabilities and which devices are connected. Do not have this add-on **Start on boot** as it will interfere with the integration. 
```